### PR TITLE
feat: add sfits to organizations

### PR DESCRIPTION
**Pull request by fschaeffler**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: src/website/content/images/organisations/sfits.svg [hunk 1]
```diff
@@ -0,0 +1 @@
... 4.6 0 01-.45 4.33c-.46.42-.86.92-1.38 1.25-3.37 2.14-7.35 2.78-8.91-1.17z"/><path d="M18.26 22c1.12.12 2.25-.36 3.42-.63 2.34-.54 4.01.31 4.9 2.48 1.77 4.33 1.01 5.91-3.47 7.27-3.37 1.02-5.41-.11-6.36-3.55-1.08-3.89-3.61-5.33-7.61-4.33-1.01.25-2.04.53-3.07.59-2.11.13-3.31-.88-3.75-2.95-.8-3.75.21-5.29 4.01-6.09.66-.14 1.34-.25 1.99-.44 2.96-.84 4.02-2.73 3.23-5.69-.55-2.08.26-3.43 2.38-3.91.71-.16 1.44-.27 2.17-.33 2.9-.26 4.43.98 4.84 3.86.2 1.42-.52 2.32-1.77 2.77-.93.33-1.92.48-2.87.74-2.79.74-4.12 2.37-3.85 4.69.41 3.48 2.17 5.38 5.09 5.52.19.01.4 0 .72 0zM31.63 32.78c-.15-1.9.7-3.16 2.22-3.75.78-.3 1.59-.5 2.38-.78a3.93 3.93 0 002.76-3.64c.23-2.78-2.44-6.4-5.12-7.12a5.54 5.54 0 00-3.23.29c-4.52 1.46-6.14-1.27-6.39-4.7 0-1.58.82-2.05 1.6-2.33 1.17-.42 2.42-.68 3.67-.71 1.37-.03 2.4.65 3.08 1.81.29.49.49 1.04.74 1.55 1.23 2.59 3.76 3.9 6.46 3.05 4.32-1.37 6.01 1.32 7.07 4.19.66 1.8-.27 3.19-2.45 3.76-1.82.48-3.15 1.34-3.61 3.25a5.37 5.37 0 00.48 3.5c.33.76.7 1.52.98 2.3.79 2.26-.06 4.18-2.27 5.11-.77.32-1.57.6-2.38.77-1.61.32-3.05-.09-4.04-1.44a11.07 11.07 0 01-1.95-5.11z" class="st1"/><path d="M33.57 37.91a10.8 10.8 0 01-1.94-5.14c-.15-1.9.7-3.16 2.22-3.75" class="st1"/></svg>
```
Comment (jsumners) [inline src/website/content/images/organisations/sfits.svg:1]: @lmammino this logo has color. Does our website desaturate color logos? Or do they need to be added without color?

Comment (lmammino) [reply src/website/content/images/organisations/sfits.svg:1]: sorry for the late reply on this. There shouldn't be any problem with colors. We do use a CSS filter to saturate the images to greyscale and then we show colors only on mouse hover

**Review by jsumners — APPROVED** [review]
LGTM

**Review by RafaelGSS — APPROVED** [review]
LGTM.

**Review by mcollina — APPROVED** [review]
lgtm

**Review by mcollina — CHANGES_REQUESTED** [review]
This looks a lot like a website of a single freelancer. I don't think we should endorse this on our website. The proper way for individuals to promote themselves is as collaborators.

**Comment by jsumners** [comment]
> This looks a lot like a website of a single freelancer. I don't think we should endorse this on our website. The proper way for individuals to promote themselves is as collaborators.

I can accept that rule.

**Comment by fschaeffler** [comment]
> This looks a lot like a website of a single freelancer. I don't think we should endorse this on our website. The proper way for individuals to promote themselves is as collaborators.

Yes, this is right. I'm a single freelancer and I'm specialised in scalable high traffic back-ends with a good focus towards AWS. For example, the last one I did that also included Fastify in some form has peaks of up to 100k RPS. Unfortunately, I'm not much of an open source contributor for Fastify as a lot of my work is under strict NDAs.

**Comment by fschaeffler** [comment]
You're totally right. This doesn't fit here as I'm not an organisation.

**Pull request closed without merging**